apps/cnspec/cmd/bundle.go

```diff
@@ -214,7 +214,7 @@ var policyPublishCmd = &cobra.Command{
 		}
 		queryHubServices, err := policy.NewPolicyHubClient(opts.UpstreamApiEndpoint(), httpClient, certAuth)
 		if err != nil {
-			log.Fatal().Err(err).Msg("could not connect to policy hub")
+			log.Fatal().Err(err).Msg("could not connect to Security Registry")
```

Comment:
The other changes use `the Mondoo Security Registry`. Should we make it consistent?

Reply:
Might as well